src/app/credExplorer/PagerankTable.js

```diff
@@ -258,3 +316,47 @@ class RecursiveTables extends React.PureComponent<RecursiveTablesProps> {
       ));
   }
 }
+
+type NeighborsTablesProps = {|
+  +neighbors: $ReadOnlyArray<Neighbor>,
+  +graph: Graph,
+  +pagerankResult: PagerankResult,
+  +depth: number,
+  +adapters: $ReadOnlyArray<PluginAdapter>,
+|};
+
+class NeighborsTables extends React.PureComponent<NeighborsTablesProps> {
+  render() {
+    const {neighbors, graph, pagerankResult, depth, adapters} = this.props;
+    return neighbors
+      .slice()
+      .sort((a, b) => {
```

Comment:
Can we use `sortBy` here so that
 1. it is simpler, and
 2. we are deterministic in case of ties on weight? (i.e., fall back to name or something)

Reply:
Done